lib/grunt.js

```diff
@@ -57,18 +57,24 @@ module.exports = function(grunt) {
     var done = this.async();
 
     let webfontsGenerator = require('webfonts-generator');
-    let iconConfig = require('../icons.json');
+    let iconConfig = JSON.parse(JSON.stringify(require('../icons.json')));
```

Comment:
during implementation I used the `../icons.json` file to pass to `--custom-json` and I noticed it would be cached if you import it twice and since I'm modifying the var it would get screwed up when it reached the custom json parsing part, and it would return my modified object.

Long story short, the stringify and parse make an effective copy of the contents of `icons.json` so it does not get screwed up when you pass the same `icons.json` to `--custom-json`. It's rare you would use `grunt --exclude-default --custom-json=./icons.json` but having it (making a copy) in there doesn't hurt.

Reply:
Should we switch to [`grunt.file.readJSON`](https://gruntjs.com/api/grunt.file#grunt.file.readjson) or even [`grunt.file.read](https://gruntjs.com/api/grunt.file#grunt.file.read) instead?

Reply:
will look into it.